minikube.yml

```diff
@@ -0,0 +1,137 @@
+# This file is for testing
+# I'm trying to deploy lifebank in MiniKube
+# Any help is welcome
```

Comment:
lets break it up into several yaml files inside a `kubernetes` folder 

see the ggoods file structure as a reference: https://github.com/eoscostarica/gGoods/tree/main/kubernetes